lib/components/util/with-suspense.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { ComponentType, Suspense } from 'react'
+
+/**
+ * Wraps a component with suspense, assuming the wrapped component is lazily loaded.
+ */
+export default function withSuspense(
+  WrappedComponent: ComponentType
+): ComponentType {
+  const suspensedComponent = (props: any): JSX.Element => (
```

Comment:
would `unknown` maybe be better here? Not sure. Or maybe you could even use a generic to use the props type from the wrapped component?

Reply:
Great idea, updated in https://github.com/opentripplanner/otp-react-redux/pull/1332/commits/f2af8414da633b2dadea7c1b53dabd2fff44ca7c